airbyte-integrations/csv/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM airbyte/base-java:dev
+COPY --from=openjdk:14.0.2-slim / /
+
+WORKDIR /airbyte
+# todo (cgardens) - naming.
+ENV INTEGRATION_NAME csv
+
+COPY build/libs/${INTEGRATION_NAME}*.jar lib/${INTEGRATION_NAME}.jar
+
+ENV DESTINATION_CLASS io.airbyte.integrations.csv.CsvDestination
+ENV DESTINATION_JAR_PATH lib/${INTEGRATION_NAME}.jar
```

Comment:
I don't think it is going to be enough. You will likely discover that with BQ. The jar won't be enough as the implementation will have external deps.

Reply:
discussed offline, agreed to just have destination have the main method which obviates all of this.